test/run

```diff
@@ -93,7 +109,8 @@ def add_e2e_tests(litani, test_dir, counter, output_dir):
             ci_stage="test",
             description=f"{test_file.stem}: init",
             outputs=run_dir / ".litani_cache_dir",
-            cwd=run_dir)
+            cwd=run_dir,
+            timeout=2 if fast else 0)
```

Comment:
I'm noticing that you setting `timeout=N if fast else 0` (where `N` is a value) for the different types of Litani jobs.

How did you determine `N`?

Does `timeout=0` cause the Litani job to not time out?

The `--timeout` section in this [page](https://awslabs.github.io/aws-build-accumulator/#litani-man_litani-add-job) does not provide a clean answer to my question.

If `timeout=0` does indeed cause the job to not time out, can we cut a short documentation issue to update this [section](https://github.com/awslabs/aws-build-accumulator/blob/release/doc/src/man/litani-add-job.scdoc#L159-L164)?

Reply:
When i tested this, `timeout=0` acted as `timeout=None` which was default.
I determined `N` by testing timeouts on jobs which i marked as `fast` and tested multiple times to come to a conclusion that the `10 jobs` with `SLOW=False` follow these timeouts.